src/Compilers/CSharp/Test/Emit3/Semantics/CollectionExpressionTests.cs

```diff
@@ -151,18 +151,18 @@ static void Main()
             if (languageVersion == LanguageVersion.CSharp11)
             {
                 comp.VerifyEmitDiagnostics(
-                    // (6,22): error CS9058: Feature 'collection expressions' is not available in C# 11.0. Please use language version 12.0 or greater.
+                    // (6,22): error CS9058: Feature 'collection expression' is not available in C# 11.0. Please use language version 12.0 or greater.
```

Comment:
The modified resource string should reflect the feature name and also is meant to be used in language version diagnostics (this one is a good example). The feature name is "collection expression**s**", therefore the string shouldn't be modified.
If we really think there is something to fix, we probably should use a different resource string in:
```
    internal partial class BoundUnconvertedCollectionExpression
    {
        public override object Display
            => (Type is null) ? MessageID.IDS_FeatureCollectionExpressions.Localize() : base.Display;
    }
```

Reply:
Thanks for the comment, Aleksey! Will try to find or add a proper one for this.